### ISSUE TITLE
Survey cell's image view issue

### ISSUE THREAD

**Issue by teamdisc** [pauloec/Nimbl3#12]
`prepareForReuse` is not implemented in `SurveyCell`. This could be used in order to return each cell back to its idle state. There's an issue with the third survey that its `imageUrl` is broken. This leads to the cell's `imageView` not able to be set the image properly, which will result in reusing the image that doesn't belong to this cell's item image.

Regarding `UIImageView` inside the cell, I think we could perhaps set its content mode to `scaleToAspectFill` in order to make it cover as a background. The current implementation has a black background color if the image isn't filled the whole screen. ✌🏻

**Comment by pauloec** [comment]
Yes, to achieve the correct UI, i should've uses `scaleToAspectFill` but then images were bad and thus, i used `UIViewContentModeScaleAspectFit`.
